tests/app/views/test_services.py

```diff
@@ -863,12 +868,13 @@ def test_should_not_index_on_service_put_if_db_exception(self):
                 content_type='application/json')
 
             assert_equal(search_api_client.index.called, False)
-            db.session.commit = Mock(side_effect=c)
             db.session.rollback()
```

Comment:
Can we do without this?  The tests pass whether it's there or not.
Since we delete everything in the database after each test, we probably don't need the rollback, right?